src/components/admin/input-error-with-character-counter/InputErrorWithCharacterCounter.module.scss

```
@@ -13,14 +13,18 @@
     @include type.small-text-regular;
     word-spacing: 0;
-    color: colors.$error;
+    color: c.$error;
     flex: 1;
     min-width: 0;
 }
 
 .counter-section {
     @include type.body-2-regular;
-    color: colors.$grey-900;
+    color: c.$grey-900;
```

Stylelint (17.2.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)